panel/1.0/troubleshooting.md

```diff
@@ -126,4 +126,11 @@ firewall-cmd --permanent --zone=trusted --change-interface=pterodactyl0
 firewall-cmd --reload
 ```
 
+If you the error `pterdactylnw already assigned to a zone` you have to add the interface to the docker zone instead
```

Comment:
This contains typos. Are you sure about the error message itself? I am certain it's `ZONE_CONFLICT: 'pterodactyl0' already bound to a zone` for which we have auto-response on our Discord server.